tests/database.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require '../database.php';
+use PHPUnit\Framework\TestCase;
+
+
+
+class Database extends TestCase{
+    public function testgenerateHtml(){
```

Comment:
You might want to do camel case? testGenerateHtml or even testSuccessGenerateHtml